daemon.go

```diff
@@ -95,24 +107,34 @@ func (d *Daemon) DialNodeConsole(label string, token *Token) (io.ReadCloser, err
 	return node.OBM.DialConsole()
 }
 
+func (d *Daemon) PowerOnNode(label string, token *Token) error {
+	err := d.usingNodeWithToken(label, token, func(n *Node) error {
+		return n.OBM.PowerOn()
```

Comment:
took me way longer than I'd like to admit to wrap my head around this syntax; still getting used to it.